Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN pnpm i --unsafe-perm=true
 ENV HOST 0.0.0.0
 EXPOSE 3000
 
-RUN mv packages/hoppscotch-app/.env.example packages/hoppscotch-app/.env
+RUN mv packages/hoppscotch-app/.env packages/hoppscotch-app/.env
```

Comment:
Why is the rename ?

Reply:
Not sure. This was probably something during the beginning when the team started working on this and we were still figuring stuff out. I just undid the changes now.